app/assets/javascripts/discourse/app/mixins/card-contents-base.js

```diff
@@ -85,15 +90,12 @@ export default Mixin.create({
     const id = this.elementId;
     const triggeringLinkClass = this.triggeringLinkClass;
     const clickOutsideEventName = `mousedown.outside-${id}`;
-    const clickDataExpand = `click.discourse-${id}`;
```

Comment:
I can't find any usage of `.discourse-user-card` or `.discourse-group-card` in the codebase.